packages/languages/src/common/observable-output-channel.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { OutputChannel, Emitter, Event } from "./languageclient-services";
+import { injectable } from "inversify";
+
+@injectable()
+export class ObservableOutputChannelManager {
```

Comment:
Can we leave out the `Observable` prefix? It's pretty normal expose events and there is no non observable version.

Reply:
Also, I think this stuff belongs into the `output-view` package. Other uses of the output facility should not need a dep to languages.

Reply:
I thought so as well, but we have a dependency from outline -> languages and this would introduce the other direction. Ideas welcome.

Reply:
I think the dependency should go `languages`-> `output`. I see the output as something very fundamentyl. Also it doesn't have a huge footprint on the UI, so I think it is fine to include it.

Reply:
`OutputChannel` from `languageclient-services` can be implemented in `languages` with our `OutputChannelManager`.

Reply:
Done